code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +155,37 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/M)
```

Comment:
DMDoc and no single-letter parameters, give the parameter a meaningful name

Reply:
Changes applied.